src/domain/space/layout/tabbedLayout/Tabs/SpaceSubspacesPage.tsx

```
@@ -97,54 +96,52 @@ const SpaceSubspacesPage = () => {
+      childEntityOnCreate={() => setCreateDialogOpen(true)}
+      createSubentityDialog={
+        <SubspaceCreationDialog
+          open={isCreateDialogOpen}
+          icon={<SpaceL1Icon2 fill="primary" />}
+          journeyName={t('common.subspace')}
+          onClose={() => setCreateDialogOpen(false)}
+          onCreate={handleCreate}
+          formComponent={CreateSubspaceForm}
+        />
+      }
+      children={
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)